bin/rspecq

```diff
@@ -95,19 +106,25 @@ opts[:max_requeues] ||= Integer(ENV["RSPECQ_MAX_REQUEUES"] || DEFAULT_MAX_REQUEU
 raise OptionParser::MissingArgument.new(:build) if opts[:build].nil?
 raise OptionParser::MissingArgument.new(:worker) if !opts[:report] && opts[:worker].nil?
 
+redis_opts = {
+  host: opts[:redis_host]
+}
+redis_url = ENV["RSPECQ_REDIS_URL"] || opts[:redis_url]
```

Comment:
For consistency, let's do what we actually do with the other options (see lines 97-104). That would be:

```ruby
opts[:redis_url] ||= ENV["RSPECQ_REDIS_URL"]

# ...
redis_opts = { host: opts[:redis_host] }
redis_opts[:url] = opts[:redis_url] if opts[:redis_url]
```